examples/ioctl.rs

```diff
@@ -0,0 +1,209 @@
+// This example requires fuse 7.11 or later. Run with:
+//
+//   cargo run --example ioctl --features abi-7-11 /tmp/foobar
```

Comment:
In the Cargo.toml file, you should add this as a `required-features` like some of the other examples

Reply:
Done.